packages/kamino-sdk/src/Kamino.ts

```diff
@@ -1797,6 +1797,17 @@ export class Kamino {
 
     return amountsSlippage;
   }
+
+  async getCollateralInfo(): Promise<CollateralInfo[]> {
+    const collateralInfos = await CollateralInfos.fetch(
+      this._connection,
+      new PublicKey('3v6ootgJJZbSWEDfZMA1scfh7wcsVVfeocExRxPqCyWH')
```

Comment:
should we set it as one of Kamino's fields so we don't hardcode the pubkey here and we can change it in the tests?